Resources/translations/SonataDashboardBundle.en.xliff

```diff
@@ -214,6 +214,22 @@
                 <source>form.label_template</source>
                 <target>Template</target>
             </trans-unit>
+            <trans-unit id="53" resname="form.label_title">
```

Comment:
Don't forget the other translation files! 

Please use the same `target` translation if you don't know the correct translation.